lfs.c

```diff
@@ -4911,3 +4911,124 @@ int lfs_migrate(lfs_t *lfs, const struct lfs_config *cfg) {
 }
 
 #endif
+
+#if LFS_THREAD_SAFE
+
+#define CREATE_LFS_TS_1(ret, function, b_type, b) \
```

Comment:
I would prefer these to not be macro wrapped, I understand the motivation but they hurt debugability. It's tedius but I think it would be best to write these out individually.

I can expand these if you're ok with that. Vim macros makes it relatively painless.

Reply:
I have them expanded in a previous commit. I'll grab those changes and update.